harness/determined/pytorch/_pytorch_context.py

```diff
@@ -1,14 +1,27 @@
 import contextlib
 import logging
-from typing import Any, Callable, Dict, Iterator, List, Optional, Set, Tuple, Type, Union, cast
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Generator,
+    Iterator,
+    List,
+    Optional,
+    Set,
+    Tuple,
+    Type,
+    Union,
+    cast,
+)
 
 import torch
 import torch.nn as nn
 
 import determined as det
 from determined import profiler, pytorch, util
 from determined.horovod import hvd
-from determined.tensorboard import get_base_path
+from determined.tensorboard import MetricWriter, get_base_path
```

Comment:
I know the previous code already violated this guideline but
https://google.github.io/styleguide/pyguide.html#22-imports

this would mean adding `tensorboard` to `from determined import ...` line.

Reply:
Updated this